Vagrantfile.centos7

```diff
@@ -51,9 +51,9 @@ EOF
     # Add a user for rootless tests
     useradd -u2000 -m -d/home/rootless -s/bin/bash rootless
 
-    # Add busybox for libcontainer/integration tests
-    . /vagrant/tests/integration/multi-arch.bash \
-        && mkdir /busybox \
-        && curl -fsSL $(get_busybox) | tar xfJC - /busybox
+    systemctl start docker
+    mkdir /busybox /debian
+    docker export $(docker create busybox) | tar -C /busybox/ -xvf -
+    docker export $(docker create debian) | tar -C /debian/ -xvf -
```

Comment:
Is there a way to maybe use something lighter than docker to just download/extract images from the repo? I frankly don't know

Reply:
there's https://raw.githubusercontent.com/moby/moby/master/contrib/download-frozen-image-v2.sh script

Reply:
https://github.com/containers/skopeo

Reply:
It's just I don't like much the idea of needing to run dockerd inside vagrant for the sole purpose of obtaining busybox (or the like) image.

What needs to be done instead is to download the images before running vagrant. Something like (in .travis.yml):
```diff
        - ln -sf Vagrantfile.fedora32 Vagrantfile
+       - make download-test-images
        - sudo vagrant up && sudo mkdir -p /root/.ssh && sudo sh -c "vagrant ssh-config >> /ro
```

(and then `vagrant up` will copy those as it copies everything from the current directory into /vagrant

Reply:
skopeo + umoci is what #2486 uses and (personal biases aside) I would much prefer that over using Docker just for the purposes of downloading like 3 images.

Reply:
I also want us to remove the need for three different images for our tests. Since it looks like folks want super esoteric architectures, we can just use Debian in #2486 and work around the missing packages.

Reply:
@cyphar 
I don't know umoci can  or can not download images for the CPU architecture supporting debian images. But umoci only support the ARM64 and AMD64 in the view of runc master branch. 
At first, I only want to make the test cases run on more CPU architecture.

Reply:
skopeo does the downloading and it does support multi-arch. umoci is a Go program and doesn't have any real architecture restrictions (aside from "you must be able to build a Go program on it"). Heck, you can even run umoci on MacOS.

All I'm suggesting is that this:

```
% docker export $(docker create busybox) | tar -C /busybox/ -xvf -
```

Becomes this:

```
% skopeo copy docker://busybox:latest oci:/tmp/busybox-image:latest
% umoci raw unpack --image /tmp/busybox-image:latest /busybox
```

That's it. And the latter will be more performant because `docker export $(docker create) | tar xvf -` requires extracting and generating tar archives multiple times (not to mention transferring the archive and all the rest of it)!

Reply:
OK, I got it.